templates/menu.inc

```diff
@@ -3,8 +3,8 @@
   <a class="nav-item nav-link ml-3" href="/contributing">contributing</a>
   <a class="nav-item nav-link ml-3" href="/downloads">downloads</a>
   <a class="nav-item nav-link ml-3" href="/documentation">documentation</a>
-  <a class="nav-item nav-link ml-3" href="http://bugzilla.gdcproject.org">bugs</a>
-  <a class="nav-item nav-link ml-3" href="https://github.com/D-Programming-GDC/GDC">current&nbsp;git</a>
+  <a class="nav-item nav-link ml-3" href="https://gcc.gnu.org/bugzilla/buglist.cgi?quicksearch=gdc">bugs</a>
```

Comment:
```suggestion
  <a class="nav-item nav-link ml-3" href="https://gcc.gnu.org/bugzilla/">bugs</a>
```
If you really insist on only listing gdc-specific bugs, then use the `[openbugs]` URL I suggested below.

Reply:
k changed